packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -119,6 +119,14 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
 
         const handleRemoveItem = () => onRemoveItem(index);
 
+        const handleClick = src => {
+            const {onItemClick} = this.props;
+
+            if (onItemClick) {
+                onItemClick(src);
```

Comment:
```suggestion
                onItemClick(option);
```

Reply:
option or option.uri?

Reply:
You should always pass the option itself out of the select box, as it cannot know whether an option has an uri and the ReferenceEditor knows more about its options.